apps/cow-fi/pages/learn/articles/[articleSlug].tsx

```diff
@@ -12,7 +15,38 @@ export interface BlogPostProps {
 }
 
 export default function BlogPostPage({ article }: BlogPostProps) {
-  return <ArticleContent article={article} />
+  const { id } = article
+  const { title, description, slug, seo } =
+    article?.attributes || {}
+  const { metaTitle, shareImage, metaDescription } = seo || {}
+  const shareImageUrl = shareImage?.data?.attributes?.url
+
+  // TODO: Add SEO information
```

Comment:
TODO

Reply:
Yes, I need to add data and add this to the meta-data, thanks for the reminder